acceptance/bundle/scripts/test.toml

```diff
@@ -1 +1,43 @@
 LocalOnly = true  # Deployment currently fails when run locally; once that is fixed, remove this setting
+
+# This test checks whether the scripts are executed in the correct order.
```

Comment:
It is not clear how having the stubs below checks that "scripts are executed in the correct order". Also, why is it important to test that scripts are executed in the correct order? It's not really a risk with scripts implementation that they will run out of order.

Reply:
No longer applicable. Been removed from the PR.